# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/mips64.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

This is just a stub patch for now to inspire others to continue porting,
and tweaking the assembly from 32 to the n64 ABI, ... and actually test it, ...

The patch depends on: cp -ar mips{,64}

  - Rene Rebe <rene@exactcode.de>

diff -u dietlibc-0.32/mips/Makefile.add dietlibc-0.32/mips64/Makefile.add
--- dietlibc-0.32/mips64/Makefile.add.vanilla	2006-06-21 20:06:30.000000000 +0000
+++ dietlibc-0.32/mips64/Makefile.add	2010-09-25 11:42:24.447227563 +0000
@@ -10,5 +10,5 @@
 ENDIAN=-meb
 endif
 CFLAGS+=$(ENDIAN) -Os -G 0 -fstrict-aliasing -fno-pic -mno-abicalls
-VPATH:=mips:syscalls.s:$(VPATH)
-LIBOBJ+=$(OBJDIR)/pread64.o $(OBJDIR)/pwrite64.o $(OBJDIR)/divdi3.o
+VPATH:=mips64:syscalls.s:$(VPATH)
+LIBOBJ+=$(OBJDIR)/pread64.o $(OBJDIR)/pwrite64.o # $(OBJDIR)/divdi3.o
Only in dietlibc-0.32/mips: divdi3.c.vanilla
